geode-core/src/main/java/org/apache/geode/internal/cache/tier/sockets/ServerHandshakeProcessor.java

```diff
@@ -71,7 +71,7 @@
 
   protected static final byte REPLY_INVALID = (byte) 61;
 
-  public static Version currentServerVersion = Acceptor.VERSION;
+  public static final Version currentServerVersion = Acceptor.VERSION;
```

Comment:
I thought there were some tests that messed with this

Reply:
Hmm, if so they must have been removed or modified.  This is building fine on both open and closed after this change.